recipes-core/rcu-hostname/files/rcuhostname

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+rcuserialpath=/usr/bin/rcu-serial
```

Comment:
Can you put a TODO comment here indicates that this script/utility hasn't ready at the moment?